### DISCUSSION TITLE
TV Brasil 2

### DISCUSSION THREAD

**Discussion by eltoncomp** [iptv-org/community#709] [Q&A]
**Channel Name:** TV Brasil 2
**Country:** Brazil
**Language:** Portuguese
**Website:** https://tvbrasil.ebc.com.br
**Stream URL (optional):** https://play.ebc.com.br/tvs
**Notes (optional):** It is the secondary channel of TV Brasil (formerly it was the government channel NBR).

**Comment by Sphinxroot** [comment]
https://www.dailymotion.com/video/x7wur7g  private video  + geo

**Comment by freearhey** [comment]
According to our database this channel was closed on 25 July 2023.

https://iptv-org.github.io/channels/br/TVBrasil2
